### ISSUE TITLE
Not enough information on pre-tokenized dataset.

### ISSUE THREAD

**Issue by rumbleFTW** [axolotl-ai-cloud/axolotl#1661]
### What piece of documentation is affected?

https://openaccess-ai-collective.github.io/axolotl/docs/dataset-formats/tokenized.html

### What part(s) of the article would you like to see updated?

There is no examples on finetuning with a pretokenized dataset. The only thing mentioned in the doc is: `Columns in Dataset must be exactly input_ids, attention_mask, labels`. But that raises these quetions:
- Should the values be pre-padded?
- What should be the ignore index for the labels?

### Acknowledgements

- [X] My issue title is concise, descriptive, and in title casing.
- [X] I have searched the existing issues to make sure this feature has not been requested yet.
- [X] I have provided enough information for the maintainers to understand and evaluate this request.

**Comment by josharian** [comment]
> What should be the ignore index for the labels?

https://github.com/OpenAccess-AI-Collective/axolotl/blob/main/docs/input_output.qmd appears to answer this:

> When the label is `-100` then that token is ignored for training.
